deploy-aks.md

```diff
@@ -140,15 +139,12 @@ The Riva Speech Skills Helm chart is designed to automate deployment to a Kubern
     kubectl get nodes "-o=custom-columns=NAME:.metadata.name,GPU:.status.allocatable.nvidia\.com/gpu"
     ```
 
-4. Ensure you are in a working directory with `riva-api` as a subdirectory, then install the Riva Helm chart. You can explicitly override variables from the `values.yaml` file, such as the `riva.speechServices.[asr,nlp,tts]` settings.
```

Comment:
Instead of removing this line, we can change to:

`You can explicitly override variables from the `values.yaml` file, such as the `modelRepoGenerator.modelDeployKey` settings.`